diff --git a/node_modules/react-codemirror2/index.js b/node_modules/react-codemirror2/index.js
index d109de4..ae47c63 100644
--- a/node_modules/react-codemirror2/index.js
+++ b/node_modules/react-codemirror2/index.js
@@ -63,7 +63,7 @@ exports.UnControlled = exports.Controlled = void 0;
 
 var React = require('react');
 
-var SERVER_RENDERED = typeof navigator === 'undefined' || global['PREVENT_CODEMIRROR_RENDER'] === true;
+var SERVER_RENDERED = typeof navigator === 'undefined';
 var cm;
 
 if (!SERVER_RENDERED) {
